Commands/ShowRecipeCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using WPF_Restaurant.Models;
+using WPF_Restaurant.Stores;
+using WPF_Restaurant.ViewModels;
+using WPF_Restaurant.Views;
+
+namespace WPF_Restaurant.Commands
+{
+    public class ShowRecipeCommand : BaseCommand
+    {
+        private MainChefViewModel _mainChefViewModel;
+        private readonly Restaurant _restaurant;
+
+        public ShowRecipeCommand(MainChefViewModel mainChefViewModel, Restaurant restaurant)
+        {
+            _mainChefViewModel = mainChefViewModel;
+            _restaurant = restaurant;
+        }
+
+        public override void Execute(object? parameter)
+        {
+            try
+            {
+                var values = (object[])parameter;
+
+                var dishId = (int)values[0];
+                var orderId = (int)values[1];
+
+                var chosenDish = _mainChefViewModel.Orders.SelectMany(x => x.OrderItems.Where(oi => oi.Id == dishId && oi.OrderNumber == orderId)).FirstOrDefault();
```

Comment:
We have orderId and dishId, so
1. Filter orders `_mainChefViewModel.Orders.SignleOrDefault(....)`
2. Then dishes in order `?.OrderItems.FirstOrDefault(...)`
Correct?

Reply:
`chosenDish` variable is getting the right dish from the list of orders